src/pages/RestaurantsPage/Buttons.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+export default function Buttons({ buttons, handleClick, selected }) {
+  return (
+    <ul>
+      {buttons.map(({ id, name }) => (
+        <li key={id}>
+          <button
+            type="button"
+            onClick={() => handleClick(id)}
+          >
+            {name}
+            {selected ? (
+              <>
+                {id === selected.id ? '(V)' : null}
+              </>
+            ) : null}
```

Comment:
jsx에서 undefined, null, false 등등은 렌더 되지 않으므로 다음과 같이 작성해도 될 것 같습니다
```suggestion
            {selected && (
              <>
                {id === selected.id ? '(V)' : null}
              </>
            )}
```

See also
  - https://reactjs.org/docs/jsx-in-depth.html#booleans-null-and-undefined-are-ignored